### PR TITLE
feat: Improvements in ff4j properties microfrontend

### PR DESCRIPTION
**Pull request by dECRISES**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # #277

## Type of change

Please check options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by paul58914080** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ karma.conf.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/paul58914080/ff4j-web-console/pullRequest?prid=5263899&bid=17244975)

File: karma.conf.js [hunk 1]
```diff
@@ -1,19 +1,24 @@
 // Karma configuration file, see link for more information
 // https://karma-runner.github.io/1.0/config/configuration-file.html
 
-module.exports = function(config) {
+module.exports = function (config) {
   config.set({
     basePath: '',
-    frameworks: ['jasmine', '@angular-devkit/build-angular'],
+    frameworks: ['parallel', 'jasmine', '@angular-devkit/build-angular'],
```
Comment (paul58914080) [inline karma.conf.js]: This is nice 👍 Does it mean we can run tests in parallel ?

File: package.json [hunk 2]
```diff
@@ -22,21 +22,25 @@
   "scripts": {
     "ng": "ng",
     "start": "run-p mock:server 'ng serve'",
+    "start:ff4j-properties": "run-p mock:server 'ng serve ff4j-properties -- --port=4202'",
```
Comment (paul58914080) [inline package.json]: Nice work. Do we need `--` twice ?

**Comment by paul58914080** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones removed
==============
+ karma.conf.js  -1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/paul58914080/ff4j-web-console/pullRequest?prid=5263899&bid=17244975)

File: package.json (hunk 2 above)
Comment (dECRISES) [reply package.json]: yes Paul, it was not taking the specified port.
We can use placeholders to give the arguments preceded by -- to scripts.
$ run-p "start-server -- --port {1}" -- 8080

Comment (paul58914080) [reply package.json]: ok

File: karma.conf.js (hunk 1 above)
Comment (dECRISES) [reply karma.conf.js]: I checked that for running test cases of sub projects, it's not running separately. Created 3 parallel instance.
With time we can observe the performance. Currently there is one 1 test case in each sub project so no much impact

**Comment by paul58914080** [comment]
Thanks a lot @dECRISES for your contribution.

**Pull request merged**